tests/drivers/touch_dev/README.md

```diff
@@ -0,0 +1,34 @@
+# About
+
+This is a manual test application for touch device drivers using the generic
+touch device API. Which touch device driver is used is determined by the board
+definition.
+
+# Usage
+
+This test application initializes the touch device and then waits
+for touch events by using interrupts by default. When touch events occur,
+the application generates output like the following:
+```
+Event: pressed!
+X: 157, Y:152
+X: 158, Y:152
+X: 158, Y:152
+X: 158, Y:152
+X: 158, Y:152
+X: 158, Y:152
+X: 158, Y:152
+X: 158, Y:152
+X: 158, Y:152
+X: 158, Y:152
+Event: released!
```

Comment:
This output doesn't correspond to the current state of this PR: by unlocking the mutex from the interrupt callback, only the position of the first pressed event is printed. This is because the mutex_lock call in the while loop blocks until the next interrupt occurs.

I'm fine with the behavior update of the application but the output in the README should be consistent with it.

Reply:
It depends. For FT5x06 based touch displays (tested with `stm32f746g-disco`), the output is exactly that one described in the `README.md` files. Once a touch is present, FT5x06 asserts an interrupt periodically. For STMPE811 you are right. But with PR #19885 the output for STMPE811 based touch displays will be similar. As long as a touch is present, it asserts an interrupt if the position changes.